apps/backend/db_patches/0171_MigrateRedeemCodesToInvites.sql

```diff
@@ -0,0 +1,70 @@
+DO
```

Comment:
This patch will allow seamless transition between Redeem codes and User Invites by preserving the existing redeem codes in the system. Only redeem-codes that are not redeemed will be transferred